packages/lib/zuauth/package.json

```diff
@@ -30,7 +30,7 @@
   ],
   "scripts": {
     "lint": "eslint \"**/*.ts{,x}\"",
-    "build": "tsc -b tsconfig.esm.json",
+    "build": "tsc -b tsconfig.esm.json tsconfig.cjs.json",
```

Comment:
do we need to make an equivalent change anywhere else?

Reply:
No. This was something I took out during development and forgot to put back in.